debian/Makefile

```diff
@@ -0,0 +1,101 @@
+FILES = src/LercLib/BitMask\
+	src/LercLib/RLE\
+	src/LercLib/BitStuffer2\
+	src/LercLib/Huffman\
+	src/LercLib/Lerc2\
+	src/LercLib/Lerc\
+	src/LercLib/Lerc_c_api_impl\
+	src/LercLib/Lerc1Decode/BitStuffer\
+	src/LercLib/Lerc1Decode/CntZImage
+
+CFLAGS = -Wall -std=c++11 -fPIC -O2 -c
+LDFLAGS = -shared -fvisibility=hidden -fvisibility-inlines-hidden
+
+RM = rm -f
+INSTALL ?= install
+INSTALL_DATA ?= $(INSTALL) -m 644
+INSTALL_LIB ?= $(INSTALL) -m 755
+
+DESTDIR     ?=
+PREFIX      ?= /usr/local
+EXEC_PREFIX ?= $(PREFIX)
+LIBDIR      ?= $(EXEC_PREFIX)/lib
+INCLUDEDIR  ?= $(PREFIX)/include
+LERC2SUFFIX ?= Lerc
+LERC1SUFFIX ?= Lerc
```

Comment:
@tmaurer3 Do you have any opinion here?
There are not any configuration for installation in current project, so I assumes include headers placed in `/usr/include/Lerc/` because headers does not have consistent prefix such as lerc_ that requires separation from other libraries. You have an option to select name for Lerc2 and Lerc1.

Reply:
@miurahr 
_ on debian in top directory: hm, ok, if you say so. 
_ makefile: if it is general Linux makefile, not specific to debian, then why not put it in build/Linux/ . 
_ include header files: both .h include files start with "Lerc_". Up to you if you want to put (copy) them into a separate folder or not. 
Once you are done with last changes (if any) let me know, then I'll merge the PR.

Reply:
> _ include header files: both .h include files start with "Lerc_". Up to you if you want to put (copy) them into a separate folder or not.

It may be not only a local decision for debian package script also affected to other projects.
 I plan to implement CMake module to detect lerc library and include file, it assumes "Lerc" folder. And  I will port it and propose it to GDAL project and vcpkg project, as a standard path for lerc.